package.json

```diff
@@ -18,6 +18,7 @@
   },
   "scripts": {
     "test": "jest --coverage && yarn run compile",
+    "test:ci": "jest",
```

Comment:
should we keep `--coverage` here so that the codecov bot we want to add can pick up the coverage?

Reply:
I'll add --cooverage for codecov separately. Think we souldn't calculate in on every test run. Configuration example: https://github.com/vlad-yakovlev/raxee-bot/tree/main/.github/workflows